src/xcode/ENA/ENA/Source/Services/HealthCertificate/Models/HealthCertifiedPerson.swift

```diff
@@ -30,7 +30,7 @@ class HealthCertifiedPerson: Codable, Equatable, Comparable {
 	required init(from decoder: Decoder) throws {
 		let container = try decoder.container(keyedBy: CodingKeys.self)
 
-		healthCertificates = try container.decode([HealthCertificate].self, forKey: .healthCertificates)
+		healthCertificates = try container.decode([HealthCertificateDecodingContainer].self, forKey: .healthCertificates).compactMap { $0.healthCertificate }
```

Comment:
Is this backwards compatible? Is it possible to encode a type of `[HealthCertificate]` and then decode a type of `[HealthCertificateDecodingContainer]`

Reply:
Yes, because what we're storing is basically just `[{base45: "HC1:xxx"}, {base45: "HC1:yyy"}]`, so as long as we keep the same property names (or rather implicit `CodingKeys` in this case) it decodes just fine.

Reply:
Ok, I See. And because `validityState` is optional it will just be nil. 👍